server/image_status.go

```diff
@@ -9,8 +9,20 @@ import (
 // ImageStatus returns the status of the image.
 func (s *Server) ImageStatus(ctx context.Context, req *pb.ImageStatusRequest) (*pb.ImageStatusResponse, error) {
 	logrus.Debugf("ImageStatus: %+v", req)
-	// TODO
-	// containers/storage will take care of this by looking inside /var/lib/ocid/images
-	// and getting the image status
-	return &pb.ImageStatusResponse{}, nil
+	image := ""
```

Comment:
should we check `image == nil` and error out if that's the case? instead of leaving to `RemoveImage`